### ISSUE TITLE
Two unclosed html tags in a markdown cell makes notebook disappear

### ISSUE THREAD

**Issue by miduncan** [nteract/nteract#5149]
**Application or Package Used**
nteract desktop, nteract web

**Describe the bug**
Two unclosed html tags in a markdown cell makes notebook disappear

**To Reproduce**

1. Open nteract desktop
2. In a new notebook, create a markdown cell
3. Type "</br></br>"
4. Click off cell to unfocus
5. Notebook will disappear 

**Expected behavior**
Either don't show invalid HTML or show as text

**Screenshots**
After following repro steps, notebooks dissappears like below
![image](https://user-images.githubusercontent.com/55718965/84080662-5f278900-a991-11ea-97d3-e5a43b845052.png)

**Desktop (please complete the following information):**

- OS: Windows 10
- Browser Firefox, Chrome

**Additional context**
Also works with "</img></img>", seems to be two closing tags of any HTML tag the markdown supports

**Comment by petebryan** [comment]
This issue is not isolated to "</br></br>" at the end of the cell. Simply having two "</br>" tags anywhere in a markdown cell causes the same issue. For example:
"Print </br>
Print </br>
Print"

**Comment by ramantehlan** [comment]
Thank you for raising this issue @miduncan and @petebryan for the observation. 

I was able to reproduce this issue. From my initial testing, this is what I found.

- It happens only with HTML tags which don't require closing tags(void elements). Ie. `</br>`, `</img>`. Even then it hasIdeally, with single tags, you should use `/` in the end, but this shouldn't break the application nonetheless.  some exceptions like `</hr></hr>` doesn't break it.
- It happens only if you are using `/` at the start of the tag instead of in the end. So `</br>` will break it, but `<br/>` won't. 
- If the cell starts with other tags, it won't break.
```
// This will not break
<h1> Test </h1>
</br></br>

// This will break
</br></br>
<h1> Test </h1>
```

@captainsafia, This seems to be an issue in markdown package, we can probably transfer it there.

**Comment by captainsafia** [comment]
Yeah, this should be transferred to the nteract/markdown repo. I'll do that now.

@ramantehlan Your analysis of this seems correct. I'm unable to reproduce this in a standard react-markdown component so I suspect that this error is caused by one of our custom parsers (either the LaTeX renderer or the code one).
